lib/chef_git/expand_node_object.rb

```diff
@@ -19,6 +19,13 @@ def message
     end
   end
 
+  def cleanup_git
+    repo = Pathname.new('/var/chef/git')
```

Comment:
Nitpick, but why use a `Pathname` here? `Dir.chdir` is happy to accept the string as well.